src/kOS/Binding/FlightControlManager.cs

```diff
@@ -487,11 +488,21 @@ private void WheelSteer(FlightCtrlState c)
             public void Dispose()
             {
                 Enabled = false;
+                if (steeringManager != null)
+                {
+                    steeringManager.RemoveInstance(shared.Vessel);
```

Comment:
Odd, I swear this was listed on the "files changed" page as still existing, but when I went to find it, I couldn't, and now it's shown as an outdated diff.  I guess it fixed itself.